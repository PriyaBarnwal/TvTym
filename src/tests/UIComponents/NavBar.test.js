
import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import NavBar from '../../UIComponents/Navbar'

test('should render navbar correctly', ()=>{
  const renderer = new ReactShallowRenderer()
  renderer.render(<NavBar/>)
  expect(renderer.getRenderOutput()).toMatchSnapshot()
})