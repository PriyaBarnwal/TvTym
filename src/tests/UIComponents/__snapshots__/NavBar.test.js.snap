// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render navbar correctly 1`] = `
<nav
  className="nav-container"
>
  <div
    className="nav-wrapper"
  >
    <ForwardRef
      className="brand-logo left"
      to="/home"
    >
      <img
        alt="logo"
        height="70"
        src="logo1.ZzdP9AdvlCYog+Quq6R6b0gN4FcLgRULcMU+jDIqD99LxUc6unyAOA=="
      />
    </ForwardRef>
    <ul
      className="right"
    >
      <li>
        <ForwardRef
          className="nav-link"
          to="/home"
        >
          <i
            className="material-icons hide-on-large-only"
          >
            home
          </i>
          <p
            className="hide-on-med-and-down"
          >
            Home
          </p>
        </ForwardRef>
      </li>
      <li>
        <ForwardRef
          className="nav-link"
          to="/favorites"
        >
          <i
            className="material-icons hide-on-large-only"
          >
            favorite
          </i>
          <p
            className="hide-on-med-and-down"
          >
            Favorites
          </p>
        </ForwardRef>
      </li>
    </ul>
  </div>
</nav>
`;
